docs/rules/no-extra-parens.md

```diff
@@ -23,6 +23,7 @@ This rule has an object option for exceptions to the `"all"` option:
 * `"conditionalAssign": false` allows extra parentheses around assignments in conditional test expressions
 * `"returnAssign": false` allows extra parentheses around assignments in `return` statements
 * `"nestedBinaryExpressions": false` allows extra parentheses in nested binary expressions
+* `"ignoreJSX": "none|all|multi-line|single-line"` allows extra parentheses around all/multi-line/single-line JSX components. Defaults to `none`.
```

Comment:
Wondering if the documentation here should say something like "no/all/multi-line/single-line" to be consistent with the available options?